### PR TITLE
[Workers for Platforms] Revamp metadata information

### PR DESCRIPTION
**Pull request by markdembo**
- Remove not-working metadata.json example
- Rename section from multipart to metadata as multipart uploads are the new default of the Getting started guide
- Add new metadata.json with exhaustive bindings examples
- Link to relevant resources in other parts of the documentation

### CODE REVIEW

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+pcx_content_type: concept
+title: User Worker Metadata
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md]:
```suggestion
title: User Worker metadata
```

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+---
+
+# User Worker Metadata
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md]:
```suggestion
# User Worker metadata
```

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+As part of the [multipart upload of user Workers](https://developers.cloudflare.com/cloudflare-for-platforms/workers-for-platforms/get-started/configuration/#4-upload-user-workers-to-a-namespace), you define the Worker's metadata. This metadata defines the Workers'configuration analogue to the [wrangler.toml file](https://developers.cloudflare.com/workers/wrangler/configuration/) for regular Workers.
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md]:
```suggestion
You define the Worker's metadata as part of the [multipart upload of user Workers](/workers-for-platforms/get-started/configuration/#4-upload-user-workers-to-a-namespace). This metadata defines the Workers'configuration analogue to the [wrangler.toml file](/workers/wrangler/configuration/) for regular Workers.
```

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      "type": "mtls_certificate",
+      "name": "<TEST_MTLS>",
+      "certificate_id": "<TEST_CERTIFICATE_ID>"
+    }
+  ]
+}
+```
+
+## Keys
+
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md:65]: Would be nice to describe the meaning of a key here, @markdembo

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+{{<Aside type="note">}}
+At a minimum, the `main_module` key is required to publish a user Worker.
+{{</Aside>}}
+
+{{<definitions>}}
+
+- `main_module` {{<type>}}strin{{</type>}} {{<prop-meta>}}required{{</prop-meta>}}
+
+  - The path to the module entrypoint of the user Worker that will be executed. For example: `main.js`.
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md]:
```suggestion
  - The path to the module entry point of the user Worker that will be executed. For example, `main.js`.
```

**Review by Maddy-Cloudflare — CHANGES_REQUESTED** [review]

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md (hunk 4 above)
Comment (markdembo) [reply content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md:65]: @Maddy-Cloudflare I reworked this a bit. Attribute is the standard term for json files.

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+      "name": "<TEST_MTLS>",
+      "certificate_id": "<TEST_CERTIFICATE_ID>"
+    }
+  ]
+}
+```
+
+## Attributes
+
+The following attributues are configurable at the top-level.
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md]:
```suggestion
The following attributes are configurable at the top-level.
```

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+- `bindings` {{<type>}}object{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+
+  - The bindings for the user Worker to enable interactions with other resources.
+
+
+{{</definitions>}}
+
+## Bindings
+
+Analogue to regular Workers, user Workers can also interact with other Cloudflare resources using Bindings. See [metadata.json example](#sample-metadata.json) and the [Wrangler configuration documentation]((https://developers.cloudflare.com/workers/wrangler/configuration/#bindings) for more information.
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md]:
```suggestion
Analogue to regular Workers, user Workers can also interact with other Cloudflare resources using Bindings. Refer to [metadata.json example](#sample-metadata.json) and the [Wrangler configuration documentation](/workers/wrangler/configuration/#bindings) for more information.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
+---
+pcx_content_type: concept
+title: User Worker metadata
+---
+
+# User Worker metadata
+
+You define the Worker's metadata as part of the [multipart upload of user Workers](/workers-for-platforms/get-started/configuration/#4-upload-user-workers-to-a-namespace). This metadata defines the Workers'configuration analogue to the [wrangler.toml file](/workers/wrangler/configuration/) for regular Workers.
```
Comment (Maddy-Cloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/platform/metadata.md]:
```suggestion
You define the Worker's metadata as part of the [multipart upload of user Workers](/cloudflare-for-platforms/workers-for-platforms/get-started/configuration/#4-upload-user-workers-to-a-namespace). This metadata defines the Workers'configuration analogue to the [wrangler.toml file](/workers/wrangler/configuration/) for regular Workers.
```

**Review by Maddy-Cloudflare — APPROVED** [review]
Fixing broken link

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**